static/css/impala/buttons.less

```diff
@@ -304,3 +304,36 @@ button[disabled],
 .button.caution.concealed {
     pointer-events: none;
 }
+.button.CTA {
+    display:inline-block;
+    *display:inline;
+    *zoom:1;
```

Comment:
Those two are for supporting old versions of IE, right? If so, could you please add a comment?